src/EditorFeatures/Core.Wpf/Completion/Presentation/RoslynCompletionSet.cs

```diff
@@ -172,13 +172,16 @@ private VSCompletion GetVSCompletion(CompletionItem item, string displayText = n
 
         public CompletionItem GetCompletionItem(VSCompletion completion)
         {
-            // Linear search is ok since this is only called by the user manually selecting 
+            // Linear search is ok since this is only called by the user manually selecting
             // an item.  Creating a reverse mapping uses too much memory and affects GCs.
             foreach (var kvp in CompletionItemMap)
             {
                 if (kvp.Value == completion)
                 {
-                    return kvp.Key;
+                    // return null if the CompletionItem.Document is null
+                    // which means it reachs the text user types on the top of the completion list in the breakpoint condition window
```

Comment:
I would mind if this comment better explained what is ultimately breaking if we don't return null here. Put another why: why is the null check here vs. whoever is calling this?